src/web-ui/public/index.html

```diff
@@ -11,6 +11,8 @@
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
     <title>Retail Demo Store</title>
+    <!-- This code is for the Segment integration - DO NOT REMOVE -->
+    <%= VUE_APP_SEGMENT_WRITE_KEY === 'NONE' ? '' : `<script>\r\n  !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error(\"Segment snippet included twice.\");else{analytics.invoked=!0;analytics.methods=[\"trackSubmit\",\"trackClick\",\"trackLink\",\"trackForm\",\"pageview\",\"identify\",\"reset\",\"group\",\"track\",\"ready\",\"alias\",\"debug\",\"page\",\"once\",\"off\",\"on\",\"addSourceMiddleware\",\"addIntegrationMiddleware\",\"setAnonymousId\",\"addDestinationMiddleware\"];analytics.factory=function(e){return function(){var t=Array.prototype.slice.call(arguments);t.unshift(e);analytics.push(t);return analytics}};for(var e=0;e<analytics.methods.length;e++){var key=analytics.methods[e];analytics[key]=analytics.factory(key)}analytics.load=function(key,e){var t=document.createElement(\"script\");t.type=\"text\/javascript\";t.async=!0;t.src=\"https:\/\/cdn.segment.com\/analytics.js\/v1\/\" + key + \"\/analytics.min.js\";var n=document.getElementsByTagName(\"script\")[0];n.parentNode.insertBefore(t,n);analytics._loadOptions=e};analytics.SNIPPET_VERSION=\"4.13.1\";\r\n  analytics.load(\"${VUE_APP_SEGMENT_WRITE_KEY}\");\r\n analytics.page();\r\n }}();\r\n<\/script>` %>
```

Comment:
What do you think about renaming `window.analytics` to `window.segmentAnalytics` or something less generic? There are a lot of "analytics" approaches and libs in the web app (Amplify, Amplitude, Optimizely, and probably Braze soon). Will eliminate possible confusion.

Reply:
I think we should avoid using the window object as much as possible because this is a bit confusing for users and also can lead to all sorts of conflicts down the road.  In this case, it is not avoidable because Segment does not yet fully support native npm libraries for tracking, though they have one in beta and when that gets a bit more robust we should move to using that.

Reply:
Got it. Does the name have to be `analytics` or can it be named something Segment-specific? I realize that this is a copy/paste from the Segment console so using a different name may create confusion with Segment's samples and docs.